client/my-sites/domains/domain-management/list/domain-item.jsx

```diff
@@ -401,9 +401,97 @@ class DomainItem extends PureComponent {
 			return;
 		}
 
-		return <div className="domain-item__meta">{ this.getSiteMeta() }</div>;
+		return (
+			<div className="domain-item__meta">
+				{ this.getSiteMeta() }
+				{ this.renderAutoRenewStatus() }
+				{ this.renderEmailStatus() }
+			</div>
+		);
 	}
 
+	renderAutoRenewStatus = () => {
+		if ( ! this.shouldShowAutoRenewStatus() ) {
+			return;
+		}
+		const autoRenewValue = this.props.domainDetails?.isAutoRenewing ? 'on' : 'off';
+		return <span className="domain-item__meta-item">Auto-renew: { autoRenewValue }</span>;
+	};
+
+	shouldShowAutoRenewStatus = () => {
+		const { domainDetails } = this.props;
+		if (
+			domainDetails?.type === domainTypes.WPCOM ||
+			domainDetails?.type === domainTypes.TRANSFER
+		) {
+			return false;
+		}
+		return ! domainDetails?.bundledPlanSubscriptionId;
+	};
+
+	renderEmailStatus = () => {
+		const { domainDetails, translate } = this.props;
+
+		if ( [ domainTypes.MAPPED, domainTypes.REGISTERED ].indexOf( domainDetails.type ) === -1 ) {
+			return;
+		}
+
+		if ( hasGSuiteWithUs( domainDetails ) ) {
+			const gSuiteMailboxCount = getGSuiteMailboxCount( domainDetails );
+
+			const text = translate(
+				'%(gSuiteMailboxCount)d mailbox',
+				'%(gSuiteMailboxCount)d mailboxes',
+				{
+					count: gSuiteMailboxCount,
+					args: {
+						gSuiteMailboxCount,
+					},
+					comment: 'The number of GSuite mailboxes active for the current domain',
+				}
+			);
+			return <span className="domain-item__meta-item">{ text }</span>;
+		}
+
+		if ( hasTitanMailWithUs( domainDetails ) ) {
+			const titanMailboxCount = getMaxTitanMailboxCount( domainDetails );
+
+			const text = translate( '%(titanMailboxCount)d mailbox', '%(titanMailboxCount)d mailboxes', {
+				args: {
+					titanMailboxCount,
+				},
+				count: titanMailboxCount,
+				comment: '%(titanMailboxCount)d is the number of mailboxes for the current domain',
+			} );
+			return <span className="domain-item__meta-item">{ text }</span>;
+		}
+
+		if ( hasEmailForwards( domainDetails ) ) {
+			const emailForwardsCount = getEmailForwardsCount( domainDetails );
+
+			const text = translate( '%(emailForwardsCount)d forward', '%(emailForwardsCount)d forwards', {
+				count: emailForwardsCount,
+				args: {
+					emailForwardsCount,
+				},
+				comment: 'The number of email forwards active for the current domain',
+			} );
+			return <span className="domain-item__meta-item">{ text }</span>;
+		}
+
+		if ( ! canCurrentUserAddEmail( domainDetails ) ) {
+			return;
+		}
+
+		return (
+			<span className="domain-item__meta-item">
+				<Button compact onClick={ this.addEmailClick } className="domain-item__email-button">
+					{ translate( 'Add email', { context: 'Button label' } ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **45** times:
`translate( 'Add email' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-item-list-updates)


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.